extensions/blocks/revue/revue.php

```diff
@@ -105,9 +106,15 @@ class="<?php echo esc_attr( $base_class . 'last-name' ); ?>"
 				</label>
 			</div>
 			<?php
-			endif;
-			// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
-			echo get_revue_button( $attributes );
+		endif;
+
+		// phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
+		if ( false !== strpos( $content, 'wp-block-jetpack-revue__fallback' ) ) {
+			echo $content;
+		} else {
+			echo get_deprecated_v1_revue_button( $attributes );
+		}
```

Comment:
Do you think you could add a comment to explain what's going on here? It took me a minute to understand where that class was coming from.